geoapi/src/main/python/opengis/metadata/acquisition.py

```diff
@@ -1,326 +1,594 @@
+# ===-----------------------------------------------------------------------===
+#    GeoAPI - Python interfaces (abstractions) for OGC/ISO standards
+#    Copyright © 2013-2024 Open Geospatial Consortium, Inc.
+#    http: //www.geoapi.org
 #
-#    GeoAPI - Programming interfaces for OGC/ISO standards
-#    Copyright © 2018-2023 Open Geospatial Consortium, Inc.
-#    http://www.geoapi.org
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
 #
+#        http: //www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+# ===-----------------------------------------------------------------------===
+"""This is the acquisition module.
+
+This subpackage contains geographic metadata structures regarding data
+acquisition that are derived from the ISO 19115-2:2019 international
+standard.
+"""
+
+__author__ = "Martin Desruisseaux(Geomatys), David Meaux (Geomatys)"
+
 
 from abc import ABC, abstractmethod
-from typing import Sequence
+from collections.abc import Sequence
+from datetime import datetime, date
 from enum import Enum
+from typing import Optional
 
+from opengis.metadata.citation import Citation, Identifier, Responsibility
+from opengis.metadata.constraints import Constraints
+from opengis.metadata.extent import Extent
+from opengis.metadata.identification import ProgressCode
+from opengis.metadata.naming import Record, RecordType
+from opengis.metadata.maintenance import Scope
 
 
 class ContextCode(Enum):
+    """
+    Designation of criterion for defining the context of the scanning
+    process event.
+    """
+
     ACQUISITION = "acquisition"
+    """Event related to a specific collection."""
     PASS = "pass"
+    """Event related to a sequence of collections."""
     WAY_POINT = "wayPoint"
-
+    """Event related to a navigational manoeuvre."""
+
+
+class EventTypeCode(Enum):
+    """Type of event related to a platform, instrument, or sensor."""
+
+    ANNOUNCEMENT = "announcement"
+    """
+    Announcementannouncement about future events relevant to the
+    platform/instrument/sensor.
+    """
+    CALIBRATION = "calibration"
```

Comment:
Maybe add a new line between enumeration values? For a non-Python developer, it is not obvious whether the documentation applies to the statement before or after the comment.